docs/docs/modeling.md

```diff
@@ -106,6 +106,46 @@ classDiagram
 
 ## ShareHouse - シェアハウス
 
+The owner can register the House owned by the owner.
+
+### ShareHouse ID (Identifier)
+
+ShareHouse ID is the UUID.
+
+### ShareHouse Name
+
+The name of the share house can set and changed. The shareHouse has the default name when the shareHouse created without specifying the name.
+
+Constraints：
+
+- At least 1 character and no more than 15 characters.
+- New ShareHouses are listed in ascending order.
+
+### rotationCycle
+
+It can select Weekly or Fortnightly.
```

Comment:
I think you should avoid using the word "select" to describe what the `rotationCycle` is.